src/App.tsx

```diff
@@ -1,14 +1,37 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter';
+import { filterTodos } from './helper';
+import { Status } from './types/Status';
 
-const USER_ID = 0;
+const USER_ID = 12051;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isError, setIsError] = useState<string | null>(null);
+  const [filterStatus, setFilterStatus] = useState<Status>(Status.All);
+
+  const showError = () => {
+    setIsError('Unable to load todos');
+    setTimeout(() => setIsError(null), 3000);
+  };
```

Comment:
```suggestion
  const showError = (message: string) => {
    setErrorMessage(message);
    setTimeout(() => setIsError(null), 3000);
  };
```
You can show other errors too in the future